dash/App/Tokens/AddToken/index.js

```diff
@@ -5,66 +5,28 @@ import chainMeta from '../../../../resources/chainMeta'
 import link from '../../../../resources/link'
 import svg from '../../../../resources/svg'
 
-class AddToken extends Component {
+class AddTokenChainScreen extends Component {
   constructor (props, context) {
     super(props, context)
-
-    this.nameDefault = 'Token Name'
-    this.symbolDefault = 'SYMBOL'
-    // this.chainIdDefault = 1
-    this.decimalsDefault = '?'
-    this.addressDefault = 'Contract Address'
-    this.logoURIDefault = 'Logo URI'
-
-    this.req = props.req || {}
-    this.token = this.req.token || {}
-    this.activeChains = props.activeChains
-
-    const chainId = parseInt(this.token.chainId)
-    const decimals = parseInt(this.token.decimals)
-
-    this.state = {
-      name: this.token.name || this.nameDefault,
-      symbol: (this.token.symbol || '').toUpperCase() || this.symbolDefault,
-      _chainId: Number.isInteger(chainId) && chainId,
-      chainId: Number.isInteger(chainId) && chainId,
-      inputAddress: (this.token.address || '').toLowerCase(),
-      address: (this.token.address || '').toLowerCase(),
-      decimals: (Number.isInteger(decimals) && decimals) || this.decimalsDefault,
-      logoURI: this.token.logoURI || this.logoURIDefault
-    }
-  }
-
-  async updateTokenData (contractAddress, chainId) {
-    const { name, symbol, decimals } = await link.invoke('tray:getTokenDetails', contractAddress, chainId)
-    this.setState({
-      name: name || this.nameDefault,
-      symbol: symbol || this.symbolDefault,
-      decimals: decimals || this.decimalsDefault
-    })
-  }
-
-  isConnectedChain () {
-    const chain = this.activeChains.find(({ id }) => id === this.state.chainId)
-    return chain.connection.primary.connected || chain.connection.secondary.connected
+    this.state = {}
   }
-
-  isDefault (statePropName) {
-    if (this.state[statePropName] === undefined) {
-      return false
-    }
-    return this.state[statePropName] === this[`${statePropName}Default`]
-  }
-
-  updateOriginChain () {
-    // const origin = this.store('main.origins', this.props.originId)
-    return (
+  
+  render () {
+    const { store } = this.props
+    const { chainId } = this.state
+    return <div className='newTokenView cardShow'>
```

Comment:
this whole class could benefit from formatting, including a space here and a couple other places

Reply:
I'll take another look briefly but we're not going hard into formatting nits because the upcoming formatting PR merge will fix them all.  I have plenty of issue with the formatting of many existing files - front-end components especially 😅